index.js

```diff
@@ -30,8 +30,9 @@ function realpathSync(filepath) {
   }
 
   const fsBinding = process.binding('fs');
+  const isWin = process.platform === 'win32';
 
-  if (fsBinding.realpath) {
+  if (fsBinding.realpath && !isWin) {
```

Comment:
I doubt this is the fix - this module exists almost entirely _because_ of windows